src/ffi/mod.rs

```diff
@@ -11,6 +11,17 @@ use tracing::{error, trace, Subscriber};
 use telio_sockets::protector::make_external_protector;
 use uuid::Uuid;
 
+#[cfg(feature = "hide_ips")]
```

Comment:
[cfg_if](https://docs.rs/cfg-if/latest/cfg_if/) is a nice way to make this a bit more clean